router-beta1-test/Chart.yaml

```diff
@@ -1,6 +1,6 @@
-name: router
+name: router-beta1-test
 home: https://github.com/deis/router
-version: 2.0.0-alpha
+version: v2-beta
```

Comment:
Good question.  I just emulated what the `workflow-beta1-test` chart had in this same field, with the assumption that it was correct.